doc/api/tracing.md

```diff
@@ -40,12 +67,125 @@ be specified with `--trace-event-file-pattern` that accepts a template
 string that supports `${rotation}` and `${pid}`. For example:
 
 ```txt
-node --trace-events-enabled --trace-event-file-pattern '${pid}-${rotation}.log' server.js
+node --trace-event-categories v8 --trace-event-file-pattern '${pid}-${rotation}.log' server.js
 ```
 
 Starting with Node.js 10.0.0, the tracing system uses the same time source
 as the one used by `process.hrtime()`
 however the trace-event timestamps are expressed in microseconds,
 unlike `process.hrtime()` which returns nanoseconds.
 
+## The `trace_events` module
+<!-- YAML
+added: REPLACEME
+-->
+
+### `Tracing` object
+<!-- YAML
+added: REPLACEME
+-->
+
+The `Tracing` object is used to enable or disable tracing for sets of
+categories. Instances are created using the `trace_events.createTracing()`
+method.
+
+#### `tracing.categories`
```

Comment:
At some point we may also want to expose the list of static categories that can be enabled by the user to facilitate interactive tool writers to expose them in the UI. With that in mind, do we want name this `Tracing.prototype.enabled_categories` so that it is more easily distinguishable from `trace_events.categories`?

Reply:
well... we can expose the list of categories that we explicitly support, but the list is not entirely static.

Reply:
re: `categories`... there is no `trace_events.categories`.

```js
const trace_events = require('trace_events');
trace_events.getEnabledCategories(); // returns the string listing of enabled categories

const tracing = trace_events.createTracing({ categories: ['foo', 'bar'] });
tracing.categories  // the string listing of categories for this tracing object
```

Reply:
Yes, there is no API for that today, but we would want one the future.

To give some more context, this is what one can do in Chrome today: 
![image](https://user-images.githubusercontent.com/79017/38389078-1fb9290e-38d2-11e8-8d74-bbff82e287d4.png)

I would imagine that people would want to build tracing tools that let users select from a list of categories of events they can trace. It would be good if such tools can obtain the predefined list of categories programmatically.

Reply:
It should be relatively trivial to add this with a separate PR once the `trace_events`  module lands.